networkd/network_monitor_thread.ml

```diff
@@ -98,6 +106,13 @@ let get_link_stats () =
 
 	let cache = Link.cache_alloc s in
 	let links = Link.cache_to_list cache in
+	let links = List.filter (fun link ->
+		let name = standardise_name (Link.get_name link) in
```

Comment:
Hmm... the above is now called twice (here and in L117). Could you factor it out using List.map?